Unix/installbuilder/datafiles/Linux.data

```diff
@@ -14,6 +14,7 @@ SERVICE_CTL:  '/opt/omi/bin/service_control'
 /opt/omi/bin/support/installssllinks;         ../scripts/installssllinks;            755; root; ${{ROOT_GROUP_NAME}}
 /opt/omi/bin/support/omid.systemd;            ../installbuilder/service_scripts/omid.systemd; 644; root; ${{ROOT_GROUP_NAME}}
 /opt/omi/bin/support/omid;                    ../installbuilder/service_scripts/omid.ulinux; 755; root; ${{ROOT_GROUP_NAME}}
```

Comment:
Making this change would require building another kit, then testing on both an init.d system and an upstart system.